Db/src/main/java/org/gusdb/fgputil/db/runner/SingleIntResultSetHandler.java

```diff
@@ -3,6 +3,8 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 
+import java.util.NoSuchElementException;
+
```

Comment:
Is this needed if only referenced in javadoc?

Reply:
It's a mystery.  It used to build before but failed on Javadoc, so yes?  They are definitely checking for legit code references now; even methods fail in @link annotations.